int32 task_id
string next_token
---
bool success